### DISCUSSION TITLE
Larger ender 3

### DISCUSSION THREAD

**Discussion by biggator32** [Jyers/Marlin#305] [Q&A] [answered]
I was Wondering if there was any way to get this awesome firmware set up for the ender 3 with the 4.2.7 bord and the bl touch that I have expanded the size to  bed size 400 by 400 by 500 tall thanks for any help

**Comment by LABMONITOR** [comment, marked as answer]
For the bed size, you'd have to change the values of ```X_BED_SIZE```, ```Y_BED_SIZE```, and ```Z_MAX_POS``` to 400, 400, and 500 respectively in ```Configuration.h``` -- as well as in your your slicer's settings.

For using the 4.2.7 board you can refer to the wiki [here](https://github.com/Jyers/Marlin/wiki/Configuring-&-Flashing); but essentially you just need to change ```#define MOTHERBOARD BOARD_CREALITY_V4``` to ```#define MOTHERBOARD BOARD_CREALITY_V427``` in ```Configuration.h``` as well.

Cheers

**Reply by biggator32** [reply]
I have done. That for what I have But how can come get all of the amazing settings they all work so well and love the screen setup

**Reply by Schnabulation** [reply]
As this firmware is open source you can just download the code, change the settings @LABMONITOR mentioned and compile it yourself. You will get all the features that come with this firmware with your own configuration changes.
